### PR TITLE
Change how nonce is generated from name

### PR DESCRIPTION
**Pull request by krzysu**
Hi, I looked into https://github.com/CirclesUBI/circles-myxogastria/issues/279 and I found that there is a better way to generate a number (nonce) from a string. I assume that should already solve most of the issues related to the same nonce generated for similar names (strings).
WDYT?

Close https://github.com/CirclesUBI/circles-myxogastria/issues/279

### CODE REVIEW

**Comment by louilinn** [comment]
Thank you @krzysu !
Looks promising! I will await one more review before merging. I will also look more closely at this during the week
I changed the base of this PR to development so we can do proper testing before releasing :)

**Comment by krzysu** [comment]
sure, let me know if there is anything else I could help with!

**Review by llunaCreixent — APPROVED** [review]
Thank you very much!

**Pull request merged**
